examples/docker-example/alchemy.run.ts

```diff
@@ -0,0 +1,100 @@
+import { alchemy } from "../../alchemy/src/alchemy";
+import { DockerImage, DockerContainer, DockerNetwork } from "../../alchemy/src/docker";
+
+// Initialize Alchemy
+const app = await alchemy("docker-example", {
+  // Determine the phase based on command line arguments
+  phase: process.argv[2] === "destroy" ? "destroy" : "up",
+  stage: process.argv[3],
+  quiet: process.argv.includes("--quiet"),
+});
+
+// Get configuration values (matching the provided Pulumi config)
+const frontendPort = 3000;
+const backendPort = 3001;
+const mongoPort = 27017;
+const mongoHost = process.env.mongoHost;
+const database = process.env.database;
+const nodeEnvironment = process.env.nodeEnvironment;
+const protocol = process.env.protocol;
+
+const stack = app.stage || "dev";
+
+// Create a Docker network
+const network = await DockerNetwork("network", {
+  name: `services-${stack}`,
+  driver: "bridge"
+});
+
+// Pull the backend image
+const backendImageName = "backend";
+const backend = await DockerImage(`${backendImageName}Image`, {
+  name: "pulumi/tutorial-pulumi-fundamentals-backend",
+  tag: "latest"
+});
+
+// Pull the frontend image
+const frontendImageName = "frontend";
+const frontend = await DockerImage(`${frontendImageName}Image`, {
```

Comment:
is it necessary to pull out `frontendImageName` as a variable name or can we inline for simplicity?

Reply:
It was a remnant from the Pulumi tutorial. I have inlined it now.